### PR TITLE
Standardized Readme

### PR DESCRIPTION
**Pull request by RichardLitt**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+Contributions welcome. Please check out [the issues](https://github.com/multiformats/c-multihash/issues).
+
+Check out our [contributing document](https://github.com/multiformats/multiformats/blob/master/contributing.md) for more information on how we work, and about contributing in general. Please be aware that all interactions related to multiformats are subject to the IPFS [Code of Conduct](https://github.com/ipfs/community/blob/master/code-of-conduct.md).
+
+Small note: If editing the Readme, please conform to the [standard-readme](https://github.com/RichardLitt/standard-readme) specification.
+
+## License
+
+[MIT](LICENSE) © Kubuxu
```
Comment (Kubuxu) [inline readme.md]: If we want to keep it under me: please change it to `© Jakub Sztandera <kubuxu@protonmail.ch>`.

Comment (RichardLitt) [inline readme.md]: Could change it to Protocol Labs Inc.?

Comment (Kubuxu) [inline readme.md]: We could, I am just not sure what we should do.
If we change it then we should also change it in LICENSE.

//cc @jbenet

Comment (jbenet) [inline readme.md]: - i think we should do `Protocol Labs, Inc` for now, and later move to `IPFS Authors` once i figure out how such an can receive copyright assignments

**Comment by Kubuxu** [comment]
Can we have the `readme.md` in `README.md`? Is this allowed by standard readme.

File: readme.md (hunk 1 above)
Comment (RichardLitt) [inline readme.md]: Changed.

**Comment by RichardLitt** [comment]
@Kubuxu You overwrote my changes. :(

**Comment by RichardLitt** [comment]
No, we can't - it must be capitalized.

**Comment by Kubuxu** [comment]
Ouch, @RichardLitt commit was `0ae7c46` you have it local on your machine.

You can fix it with `git reset --hard 0ae7c46` and then force push.

**Comment by Kubuxu** [comment]
This is way that many C projects are set up, support files are in caps, normal are not.

Now it looks weird as for example LICENSE is in caps.

**Comment by RichardLitt** [comment]
License should be in caps.

**Comment by RichardLitt** [comment]
@Kubuxu Check it now. What about install and usage?

**Comment by Kubuxu** [comment]
There is not really install process and I can write the usage as a lib later.

**Comment by RichardLitt** [comment]
OK. So... good to go?

**Comment by Kubuxu** [comment]
Yup, feel free to merge if you also think that everything is done.

**Pull request merged**